zipline/__main__.py

```diff
@@ -251,6 +251,12 @@ def run(ctx,
 def zipline_magic(line, cell=None):
     """The zipline IPython cell magic.
     """
+    load_extensions(
+        default=True,
+        extensions=[],
+        strict=True,
+        environ=os.environ,
+    )
     try:
         return run.main(
```

Comment:
I am going to merge this as is because I don't think I have time to work out a better solution for consistently setting this up. We can always clean this up in the future but this fixes our immediate issue with extensions in the notebook